### PR TITLE
Use "openstack" instead of "keystone" for LDAP test

### PR DESCRIPTION
**Pull request by rhafer**
The openstack command works with the v3 and v2 API.

### CODE REVIEW

**Comment by rhafer** [comment]
This should fix the testing for: https://github.com/crowbar/crowbar-openstack/pull/116

**Comment by vuntz** [comment]
+1

**Comment by aplanas** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3139,7 +3135,7 @@ function oncontroller_testsetup()
     done
 
     if [[ $want_ldap ]] ; then
-        keystone user-get bwiedemann | grep -q 82608 || complain 103 "LDAP not working"
+        openstack user show bwiedemann | grep -q 82608 || complain 103 "LDAP not working"
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: --insecure got lost now?!

**Pull request merged**